### PR TITLE
Fix a bug in fd.ops.squeeze.

### PR DESCRIPTION
**Pull request by wujingyue**
Could have been caught if #3117 were fixed.

### CODE REVIEW

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

File: tests/python/test_alias.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+def test_squeeze():
+    with FusionDefinition() as fd:
+        inp = fd.define_tensor([1, 1, 2], contiguity=True)
+        out = fd.ops.squeeze(inp, [0, 1])
+        out = fd.ops.slice(out, [1], [2])
+        fd.add_output(out)
+
+    inp_tensor = torch.randn(1, 1, 2, device="cuda")
+    out_tensor = fd.execute([inp_tensor])[0]
```
Comment (rdspring1) [inline tests/python/test_alias.py:18]: Is there a reason to use `exec_nvfuser` in `NVFuserTest`?

It does the serialization check and print definition check.

**Comment by wujingyue** [comment]
!build

**Review by rdspring1 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**